Sources/MapboxCoreNavigation/URLSession.swift

```diff
@@ -8,23 +8,44 @@ extension URLSession {
      The user agent string for any HTTP requests performed directly within MapboxCoreNavigation or MapboxNavigation.
      */
     public static let userAgent: String = {
+        // Bundles in order from the application level on down
+        #if SWIFT_PACKAGE
+        let bundles: [Bundle?] = [
+            .main,
+            .mapboxNavigationIfInstalled,
+            .mapboxCoreNavigation
+        ]
+        #else
         let bundles: [Bundle?] = [
-            // Bundles in order from the application level on down
             .main,
             .mapboxNavigationIfInstalled,
             .mapboxCoreNavigation,
             .init(for: Directions.self),
         ]
+        #endif
+
         let bundleComponents = bundles.compactMap { (bundle) -> String? in
             guard let name = bundle?.object(forInfoDictionaryKey: "CFBundleName") as? String ?? bundle?.bundleIdentifier else { return nil }
             var stringForShortVersion: String? {
+                #if SWIFT_PACKAGE
+                let mapboxNavigationName = "MapboxNavigation_MapboxNavigation"
```

Comment:
Should we use `"MapboxNavigation"` for the `name` part of the user agent? As for now, we have different user agents for SPM and non-SPM installations.

Reply:
Agree, changed to use default bundle names for SPM and CocoaPods.